src/screens/FeedScreen/Header.js

```diff
@@ -149,7 +149,7 @@ const _renderAnonimity = ({
       icon: <TrashRed />,
       onPress: () => {
         refSheet.current.close();
-        onDeletePost();
+        onDeletePost(item);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `onDeletePost()` to `onDeletePost(item)` is a good one if the deletion process now depends on specific properties of the `item` object. However, it's important to ensure that all calls to `onDeletePost` in your codebase are updated to pass this new argument to avoid potential errors.
```diff
- onDeletePost();
+ onDeletePost(item);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->